### PR TITLE
Creation/Management Errors and loaders display

### PR DESCRIPTION
**Pull request by ThomasBarrial**

### CODE REVIEW

**Review by 0xSyzy — APPROVED** [review]

File: components/Error.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Text, View } from 'react-native';
+import React from 'react';
+import tw from 'tailwind-react-native-classnames';
+import { ApolloError } from '@apollo/client';
+
+interface IProps {
+  // eslint-disable-next-line react/require-default-props
```
Comment (louisledoux) [inline components/Error.tsx]:
```suggestion
```

File: components/animated/SkeletonLoader.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    borderBottomWidth: 1,
+    borderColor: 'rgba(135, 144, 224, 0.84)',
+  },
+  profil: {
+    backgroundColor: '#8790E0',
+  },
+});
+
+interface ISkeletonLoader {
+  // eslint-disable-next-line react/require-default-props
```
Comment (louisledoux) [inline components/animated/SkeletonLoader.tsx]:
```suggestion
```

File: components/animated/cardsLoader/CardListLoader.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+export default function CardListLoader() {
+  return (
+    <View style={styles.container}>
+      <CardLoader />
+      <CardLoader />
+      <CardLoader />
+      <CardLoader />
+      <CardLoader />
+      <CardLoader />
```
Comment (louisledoux) [inline components/animated/cardsLoader/CardListLoader.tsx]: nani-huh

File: components/animated/newsFeedLoaders/ContainerLoader.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+export default function ContainerLoader() {
+  return (
+    <View style={styles.container}>
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
```
Comment (louisledoux) [inline components/animated/newsFeedLoaders/ContainerLoader.tsx]: ah-shit-here-we-go-again-ah-shit

File: components/animated/pageLoader/PageLoader.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        <SkeletonLoader variant="box" height={10} width="100%" />
+      </View>
+      <View style={tw`w-11/12 mt-5`}>
+        <SkeletonLoader variant="box" height={50} width="100%" />
+        <SkeletonLoader variant="box" height={50} width="100%" />
+        <SkeletonLoader variant="box" height={50} width="100%" />
+        <View style={tw`mt-5`}>
+          <SkeletonLoader variant="box" height={50} width="100%" />
+        </View>
+      </View>
```
Comment (louisledoux) [inline components/animated/pageLoader/PageLoader.tsx:51]: cat-heavy-breathing-intensifies

**Review by louisledoux — APPROVED** [review]

**Pull request merged**